pkg/services/sqlstore/migrations/query_history_star_mig.go

```diff
@@ -24,4 +24,7 @@ func addQueryHistoryStarMigrations(mg *Migrator) {
 	mg.AddMigration("add column org_id in query_history_star", NewAddColumnMigration(queryHistoryStarV1, &Column{
 		Name: "org_id", Type: DB_BigInt, Nullable: false, Default: "1",
 	}))
+	
+	mg.AddMigration("alter table query_history_star_mig column user_id type to bigint", NewRawSQLMigration("ALTER TABLE query_history_star ALTER COLUMN user_id TYPE BIGINT;"))
```

Comment:
@grafana/backend-platform should we use this instead?

```suggestion
	mg.AddMigration("alter table query_history_star_mig column user_id type to bigint", NewRawSQLMigration("").Mysql("ALTER TABLE query_history_star ALTER COLUMN user_id TYPE BIGINT;"))
```

Reply:
You'll need to make sure you get the database into the state you want it to be in for MySQL, Postgres, and SQLite. For MySQL, the format of the migration needs to be different from Postgres.

It'll probably look something like this if you don't need a migration for SQLite as well.

```suggestion
	mg.AddMigration("alter table query_history_star_mig column user_id type to bigint", NewRawSQLMigration("").Mysql("ALTER TABLE query_history_star MODIFY user_id BIGINT;")).Postgres("ALTER TABLE query_history_star ALTER COLUMN user_id TYPE BIGINT;"))
```